### ISSUE TITLE
Pane with errors disappears almost instantaneously with prettify_on_save

### ISSUE THREAD

**Issue by Pastafarianist** [SublimeHaskell/SublimeHaskell#383]
Steps to reproduce:

1. Make sure that your config file has `"prettify_on_save": true`.
2. Open some invalid Haskell file, such as

```
main =
	a <- foo
	bar a
```

3. Hit `Ctrl+S`.

You will see briefly flickering pane at the bottom of the window, disappearing after a split second. If you hit `Ctrl+S` repeatedly, you will be able to discern the contents of this pane:

```
stylish-haskell --config /path/to/.stylish-haskell.yaml failed, stderr contents:
Language.Haskell.Stylish.Parse.parseModule: could not parse <unknown>: ParseFailed (SrcLoc "<unknown>.hs" 2 11) "Parse error: <-"
```

It would be great if this pane did not disappear.

**Comment by bscottm** [comment]
@Pastafarianist: You can always call up a panel from the lower left corner -- you should see a little window or laptop-looking icon. The "Sublime Haskell Output Window" is the one that has the error/warning/hint output. There is another, "Haskell Run Output", which has the `stylish-haskell` (and other directly run command) stderr output. Not sure which one should be visible. Preferably, the SHOW panel, no the HRO panel, I would imagine.

**Comment by Pastafarianist** [comment]
This also seems to be fixed on 2.1.16.
